### ISSUE TITLE
tests: set up a11y testing

### ISSUE THREAD

**Issue by IgnaceMaes** [IgnaceMaes/ember-shiki#10]
The addon should have an automated test which validates accessibility best practices.

Related: https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/pull/112
